connectors/graphql/element-templates/template-connector.json

```diff
@@ -0,0 +1,383 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "GraphQL Connector",
+  "id": "io.camunda.connectors.GraphQL.v1",
+  "description": "Execute GraphQL query",
+  "version": 1,
+  "documentationRef": "https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/template/",
```

Comment:
This link is dead

Reply:
I missed the variables, thank you for noticing. Fixed it, and the link also. See https://github.com/camunda/web-modeler/pull/3530